config/supervisord.conf

```diff
@@ -5,6 +5,7 @@ file=/var/run/supervisor.sock
 pidfile=/var/run/supervisor.pid
 logfile=/var/log/supervisor.log
 logfile_backups=1
+environment=DEVELOPMENT=True
```

Comment:
FYI I've got a private repo where I've got a different `supervisord.conf` with the production CF rabbit and postgres credentials inlined into `environment`.  Using it so I can make AWS celery instance provisioning turn-key.  I can invite you to the repo if you want.